outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "alarm_id" {
+  value       = aws_cloudwatch_metric_alarm.success_rate[0].id
+  description = "The ID of the CloudWatch metric alarm"
+}
+
+output "alarm_arn" {
+  value       = aws_cloudwatch_metric_alarm.success_rate[0].arn
+  description = "The ARN of the CloudWatch metric alarm"
+}
+
+output "alarm_name" {
+  value       = aws_cloudwatch_metric_alarm.success_rate[0].alarm_name
```

Comment:
please use try, as module.this.enabled might be used to not create the alarm and then the outputs fail to be created.